Bottom: f8803fee89b7f1c99a09bd882074f99263035a8d
Top:    42828ec7ee9ce7510d93e16c23cd010904b52dda
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2013-06-08 21:43:56 -0400

Refresh of jds-1

---

diff --git a/devel/shellinabox/Makefile b/devel/shellinabox/Makefile
new file mode 100644
index 0000000..b963cc7
--- /dev/null
+++ b/devel/shellinabox/Makefile
@@ -0,0 +1,42 @@
+#
+# Copyright (C) 2009 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=shellinabox
+PKG_VERSION:=2.14
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://shellinabox.googlecode.com/files/
+PKG_MD5SUM:=6c63b52edcebc56ee73a108e7211d174
+
+PKG_INSTALL=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/shellinabox
+  CATEGORY:=Development
+  SECTION:=devel
+  TITLE:=shellinabox
+  URL:=http://www.shellinabox.org/
+  DEPENDS:=
+endef
+
+define Package/shellinabox/description
+	shellinabox
+endef
+
+define Build/Compile
+	$(MAKE) -C $(PKG_BUILD_DIR)
+endef
+
+define Package/shellinabox/install
+	cp -r $(PKG_INSTALL_DIR)/* $(1)
+endef
+
+
+$(eval $(call BuildPackage,shellinabox))
